app/assets/stylesheets/govuk_publishing_components/components/_big-number.scss

```
@@ -1,11 +1,12 @@
 .gem-c-big-number {
   margin-bottom: govuk-spacing(3);
-  font-family: $govuk-font-family;
+  @include govuk-typography-common($font-family: $govuk-font-family);
```

No need to pass the font family unless you want to override it for some reason.

```suggestion
  @include govuk-typography-common;
```